packages/cargo/src/generators/library/generator.ts

```diff
@@ -21,6 +21,17 @@ export default async function (host: Tree, opts: CLIOptions) {
 		projectType: "library",
 		sourceRoot: `${options.projectRoot}/src`,
 		targets: {
+      build: {
+        executor: '@nxrs/cargo:build',
+        options: {
+          release: false,
+        },
+        configurations: {
+          production: {
+            release: false,
+          },
+        },
+      },
```

Comment:
```suggestion
			build: {
				executor: '@nxrs/cargo:build',
				options: {
					release: false,
				},
				configurations: {
					production: {
						release: true,
					},
				},
			},
```
Would you mind using tabs for indentation here and setting the `production` default to `release: true`?